Description: Use global grunt
Author: Praveen Arimbrathodiyil <praveen@debian.org>
Bug: https://github.com/gruntjs/grunt-cli/pull/55
Forwarded: https://github.com/gruntjs/grunt-cli/pull/55
Reviewed-By: Yadd <yadd@debian.org>
Last-Update: 2022-01-09

--- a/bin/grunt
+++ b/bin/grunt
@@ -53,6 +53,11 @@
     Grunt.execute(env, function(env) {
       var tasks = options.argv.remain;
       delete options.argv;
+      // look for global grunt install
+      if (!env.modulePath)
+        try {
+          env.modulePath = require.resolve('grunt');
+        } catch (ex) { }
       // No grunt install found!
       if (!env.modulePath) {
         if (options.version) {
